homeworks/roman.vlasov_batteryfree/homework_3/function_array.js

```diff
@@ -0,0 +1,16 @@
+function degree(a) {
+    return a * a;
+}
+function arraystep(a, b) {
+    const arr = [];
+    for (let i = 0; i <= (parseInt(a / b, 10)); i++) {
+        arr[i] = i * b;
+    }
+    return arr;
+}
+const arr = arraystep(32, 4);
+for (let i = 0; i < arr.length; i++) {
+    arr[i] = degree(arr[i]);
```

Comment:
not the best practice for arrays, organize you code to use  .push() method, but not by index assignment